classes/book.rb

```diff
@@ -0,0 +1,17 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  # attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+    rental.person.rentals.push(rental) unless rental.person.rentals.include?(rental)
+  end
```

Comment:
- [ ] Instead of sending the rental as a parameter and setting the book as `self` afterward. Kindly use the following method of adding a rental to achieve the 'has-many' association.

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```